modules/web/client/src/main/webapp/styles/prime.scss

```diff
@@ -30,10 +30,7 @@
 
 body,
 body .p-component {
-  margin: 0;
```

Comment:
Unrelated. This was unnecessary and was interfering with the new primereact styles.